internal/orchestrion/_integration/valkey-go/valkey.go

```diff
@@ -0,0 +1,93 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2023-present Datadog, Inc.
+
+//go:build linux || !githubci
+
+package valkey
+
+import (
+	"context"
+	"strings"
+	"testing"
+
+	"github.com/DataDog/dd-trace-go/internal/orchestrion/_integration/internal/containers"
+	"github.com/DataDog/dd-trace-go/internal/orchestrion/_integration/internal/trace"
+	"github.com/stretchr/testify/require"
+	testvalkey "github.com/testcontainers/testcontainers-go/modules/valkey"
+	"github.com/valkey-io/valkey-go"
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/tracer"
+)
+
+type TestCase struct {
+	client valkey.Client
+}
+
+func (tc *TestCase) Setup(ctx context.Context, t *testing.T) {
+	containers.SkipIfProviderIsNotHealthy(t)
+	valkeyContainer, err := testvalkey.Run(ctx, "valkey/valkey:latest")
```

Comment:
```suggestion
	valkeyContainer, err := testvalkey.Run(ctx, "valkey/valkey:8-alpine")
```

lets try to pin the major version to prevent possible flakes in the future.

Also, could you add a new `docker pull` to the CI [here](https://github.com/DataDog/dd-trace-go/blob/main/.github/workflows/orchestrion.yml#L169-L188) to speed up the CI? 🙏